scripts/build.py

```diff
@@ -172,7 +172,7 @@ def cmake_version():
         r'cmake version (?P<version>[\d\.]+)', cmake_version_output)
     if match is None:
         raise BuildError('failed to get CMake version')
-    cmake_version = distutils.version.StrictVersion(match.group('version'))
+    cmake_version = packaging.version.Version(match.group('version'))
```

Comment:
This parses the version string twice; and the `cmake version` pattern will fail on many inputs legal to `packaging.version`.  `packaging.version.VERSION_PATTERN` can be used to construct a pattern that will correctly match all legal inputs and only require parsing once. 

But this may not be important if we don't need to support the full version syntax... but then, if we only need to support `X.Y.Z` versions, a tuple comparison is easier than adding a whole module just for version comparison.